### ISSUE TITLE
🐛 Bug Report: [Python SDK] Optional parameter "role" is required when creating a team

### ISSUE THREAD

**Issue by sheecegardezi** [appwrite/appwrite#5761]
### 👟 Reproduction steps

```
from appwrite.client import Client
from appwrite.services.teams import Teams

client = Client()

(client
  .set_endpoint('http://localhost/v1')
  .set_project('******')
  .set_key('*******') 
)

teams = Teams(client)

result = teams.create('admin', 'admin')

```

### 👍 Expected behavior

it should create a team in dashboard names admin. Following code works just fine notice the empty array passed for roles:

```
from appwrite.client import Client
from appwrite.services.teams import Teams

client = Client()

(client
  .set_endpoint('http://localhost/v1')
  .set_project('******')
  .set_key('*******') 
)

teams = Teams(client)

result = teams.create('admin', 'admin', [])

```

### 👎 Actual Behavior

if you dont pass empty array fo roles then it throws the following error in terminal when you run the script:

```
(.venv) user@machine:$~/.venv/bin/python ~/appwrite/startup.py

Traceback (most recent call last):
  File "~/.venv/lib/python3.10/site-packages/appwrite/client.py", line 97, in call
    response.raise_for_status()
  File "~/.venv/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://localhost/v1/teams

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "~/appwrite/startup.py", line 14, in <module>
    result = teams.create('admin', 'admin')
  File "~/.venv/lib/python3.10/site-packages/appwrite/services/teams.py", line 40, in create
    return self.client.call('post', path, {
  File "~/.venv/lib/python3.10/site-packages/appwrite/client.py", line 109, in call
    raise AppwriteException(response.json()['message'], response.status_code, response.json().get('type'), response.json())
appwrite.exception.AppwriteException: Server Error
```

in docker logs you can see the issue:

```
(.venv) user@machine:$ docker logs appwrite

[Error] Timestamp: 2023-07-03T17:02:48+00:00
[Error] Method: POST
[Error] URL: /v1/teams
[Error] Type: TypeError
[Error] Message: {closure}(): Argument #3 ($roles) must be of type array, null given, called in /usr/src/code/vendor/utopia-php/framework/src/App.php on line 593
[Error] File: /usr/src/code/app/controllers/api/teams.php
[Error] Line: 64
```

### 🎲 Appwrite version

Version 1.3.x

### 💻 Operating system

Linux

### 🧱 Your Environment

local machine

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by sheecegardezi** [comment]
I am unsure, I just use the workaround mentioned above. I can test it out over the weekend ?

**Comment by abnegate** [comment]
Closing as fixed in 1.4.x and Python SDK 4.0.0
